src/pages/admin/Announcements.scss

```diff
@@ -0,0 +1,93 @@
+.announcementContainer {
+  min-height: 500px;
+  max-height: 85vh;
+  width: 100%;
+  margin-top: 30px;
+  position: relative;
+  overflow: auto;
+}
+
+/* Announcement Page Header */
+
+.announcementHeader {
+  margin-top: 0;
```

Comment:
For all the `margin` properties, you could use the shorthand syntax. In this case `margin: 0 auto 16px`.